runtime/opt/taupage/init.d/04-configure-network-interfaces.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env python
+
+import argparse
+import logging
+
+import boto.ec2
+import boto.utils
+import time
+import sys
+import subprocess
+import netifaces
+from netaddr import IPAddress
+
+from taupage import configure_logging, get_config
+
+
+def instance_id():
+    """Helper to return theid for the current instance"""
```

Comment:
It might be a better idea to move the common code to the python package in `/usr/local/lib/python3.4/dist-packages/taupage` instead of copying it every single time.

Reply:
PR updated, thanks for this. Don't know too much about taupage and was trying to limit the amount of things I was touching with this change.